src/Cache.php

```diff
@@ -169,7 +170,7 @@ public function cache(Request $request, Response $response)
      */
     public function forget($slug)
     {
-        return $this->files->delete($this->getCachePath($slug.'.html'));
+        return $this->files->delete($this->getCachePath($slug.'.html')) || $this->files->delete($this->getCachePath($slug.'.json'));
```

Comment:
There'll be a bug here if there's the same file name with both extensions `.html` and `.json` as the first file will be deleted and the value `true` will be returned

You'll need to do something like:
```
$deletedHTMLState = $this->files->delete($this->getCachePath($slug.'.html'))
$deletedJSONState = $this->files->delete($this->getCachePath($slug.'.json'))

return $deletedHTMLState || $deletedJSONState;
```

Reply:
Hi Adrian

I agree that this could potentially be a problem. Is it though possible to get the same file name with the same extension? Since in the URL rewriting, it is not being checked, which Content-Type is being requested. When the file is being written, only the path is taken into account as well:

https://github.com/JosephSilber/page-cache/blob/master/src/Cache.php#L191

So I agree, it could potentially be a problem. The question is, if checking for the Content-Type should be handled as well or not.

Thank you for taking a look at the PR :)